lib/local-links-manager/import/services_tier_importer.rb

```diff
@@ -41,33 +41,108 @@ def import_item(item, response, summariser)
         service = Service.find_by(lgsl_code: item[:lgsl_code])
         raise MissingRecordError, "LGSL #{item[:lgsl_code]} is missing" if service.nil?
         Rails.logger.info("Updating service '#{service.label}' (lgsl #{service.lgsl_code})")
+        checked_services.add(service)
 
         if item[:tier].blank?
           response.errors << "LGSL #{item[:lgsl_code]} is missing a tier"
           summariser.increment_ignored_items_count
-        elsif not update_tier(service, item[:tier])
+        elsif not create_and_update_tiers(service, item[:tier])
           response.errors << "LGSL #{item[:lgsl_code]} has incorrect tier name"
           summariser.increment_ignored_items_count
         else
           summariser.increment_updated_record_count
         end
       end
 
-      def update_tier(service, tier_name)
+      def create_and_update_tiers(service, tier_name)
         case tier_name
         when 'district/unitary'
-          ServiceTier.create(service: service, tier_id: Tier.district)
-          ServiceTier.create(service: service, tier_id: Tier.unitary)
+          if service.tiers.include?('district')
```

Comment:
This class is modifying data related to the `Service` class a lot. Maybe indicative of [feature envy](https://sourcemaking.com/refactoring/smells/feature-envy). Could this logic live in the `Service` class instead?